client/src/pages/Projects/index.tsx

```diff
@@ -0,0 +1,28 @@
+import Page from "src/layouts/page";
+import { sampleData } from "src/entities/project.entities";
+import { RootState } from "../../store";
+import { useSelector } from "react-redux";
+import Project from "src/components/Projects/Project";
+
+const Projects = () => {
+  const { theme } = useSelector((state: RootState) => state.global);
+
+  return (
+    <Page>
+      <div className="text-3xl">
+        Explore TosiDrop Projects
+        <ul>
```

Comment:
it is better to use flexbox for space between elements. You can do

```
<div className="flex flex-col gap-4">
  { map function }
</div>
```
using this you don't need to manually add margin to each box, hence less place to change, when needed.

the standard for flex gap is :

- `gap-2`: for text, e.g., space between icon and text
- `gap-4`: space between bigger component (between buttons etc.)
- `gap-8`: space between title and page